opal-core-ws/src/main/java/org/obiba/opal/web/system/SystemResource.java

```diff
@@ -235,4 +244,12 @@ public KeyStoreResource getKeyStoreResource() {
     resource.setKeyStore(systemKeyStoreService.getKeyStore());
     return resource;
   }
+
+  @GET
+  @Path("/plugins/vcf-stores")
+  public Response getVcfStores() {
```

Comment:
Not the right place to have this entry point. There is already a PluginsResource